### PR TITLE
Ft user signup

### PR DESCRIPTION
**Pull request by Kellswork**
### What does this PR do?
I don't really know how to explain it, it contains some components I created to use so I can use Atomic Design system

###Notion Card?
It's part of the signup card

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,17 +39,20 @@
     "eslint-plugin-react-hooks": "^2.5.1",
     "html-webpack-plugin": "^4.3.0",
     "mini-css-extract-plugin": "^0.9.0",
```
Comment (omenkish) [inline package.json:41]: If we are using styled-components, do we still need this guy?

File: src/components/atoms/icons.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+export const IconRight = () => (
```
Comment (omenkish) [inline src/components/atoms/icons.js:3]: All the SVGs should have the values you fixed as props. width, height, viewBox, fill should all be dynamic so we can set them differently if needs be.

File: src/components/organisms/Forms.js [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    color: ${errorCol.error800};
+    margin-top: 0px;
+}
+`
+const Loading = styled.div`
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  align-items: center;
+`
```
Comment (omenkish) [inline src/components/organisms/Forms.js:103]: Can we move all the styled-components into a different file? Then we can just import them here

File: src/components/organisms/Forms.js [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+  password: yup
+    .string()
+    .required('Password is required')
+    .min(5, 'password must be atleast 5 characters')
+    .max(50, 'maximum of 50 characters')
+    .trim(),
+  confirmPassword: yup
+    .string()
+    .required('required')
+    .oneOf([yup.ref('password'), null], 'password does not match'),
```
Comment (omenkish) [inline src/components/organisms/Forms.js:133]: The error message should be `passwords do not match` not `password does not match`

File: src/components/organisms/Forms.js [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+}
+`
+const Loading = styled.div`
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  align-items: center;
+`
+
+const signupSchema = yup.object().shape({
```
Comment (omenkish) [inline src/components/organisms/Forms.js:105]: can we move this to another file in this same directory? We don't want this file to be unnecessarily long, do we?

**Review by omenkish — CHANGES_REQUESTED** [review]

File: src/components/organisms/Forms.js (hunk 3 above)
Comment (Kellswork) [reply src/components/organisms/Forms.js:103]: Yes

File: package.json (hunk 1 above)
Comment (Kellswork) [reply package.json:41]: It's for webpack

File: src/components/organisms/Forms.js (hunk 5 above)
Comment (Kellswork) [reply src/components/organisms/Forms.js:105]: Yes, I'll do that after implementing login

File: src/components/atoms/icons.js (hunk 2 above)
Comment (Kellswork) [reply src/components/atoms/icons.js:3]: I doubt there will be a need to change their sizes but I'll work on that too

**Pull request closed without merging**